activiti-cloud-service-common/activiti-cloud-services-swagger/src/main/java/org/activiti/cloud/common/swagger/springdoc/customizer/CustomOperationIdCustomizer.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.common.swagger.springdoc.customizer;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import org.springdoc.core.customizers.OperationIdCustomizer;
+
+public class CustomOperationIdCustomizer extends OperationIdCustomizer {
+
+    @Override
+    public void customise(OpenAPI openAPI) {
```

Comment:
After upgrading versions, `NamingOperationCustomizer` was executed before the default  Springdoc default `OperationIdCustomizer` that is adding `_1` to `operationId`. For this reason only solution I found was override the default bean